lib/dry/web/roda/generators/sub_app.rb

```diff
@@ -1,29 +1,49 @@
-require "dry/web/roda/generate"
 require "dry/web/roda/generators/inflections"
+require "dry/web/roda/generators/abstract_project"
 
 module Dry
   module Web
     module Roda
       module Generators
-        class SubApp
-          attr_reader :generate
+        class SubApp < AbstractProject
```

Comment:
In this context, I'm not sure that `AbstractProject` is the best name for this superclass, since a `SubApp` isn't actually an overall "project." Perhaps we could rename `AbstractProject` to `AbstractGenerator`?